deploy/example_clusters/cr-cluster-cdh632-1.1cm-stor.yaml

```diff
@@ -0,0 +1,122 @@
+# apiVersion: v1
+# kind: ConfigMap
+# metadata:
+#   name: ha
+#   labels:
+#     kubedirector.hpe.com/cmType : ha
+# data:
+#   ha: "true"
+# ---
+---
+apiVersion: "kubedirector.hpe.com/v1beta1"
+kind: "KubeDirectorCluster"
+metadata:
+  name: "cdh632-cm-persistent"
+spec:
+  app: cdh632
+  # connections:
+  #   configmaps:
+  #   - ha
+  roles:
+  - id: cmserver
+    resources:
+      requests:
+        memory: "18Gi"
+        cpu: "4"
+      limits:
+        memory: "18Gi"
+        cpu: "4"
+    storage:
+      size: "100Gi"
+  - id: primary
+    resources:
+      requests:
+        memory: "4Gi"
+        cpu: "4"
+      limits:
+        memory: "4Gi"
+        cpu: "4"
+    storage:
+      size: "100Gi"
+  - id: hive
+    resources:
+      requests:
+        memory: "2Gi"
+        cpu: "2"
+      limits:
+        memory: "2Gi"
+        cpu: "2"
+    storage:
+      size: "100Gi"
+  - id: worker
+    members: 3
+    resources:
+      requests:
+        memory: "4Gi"
+        cpu: "2"
+      limits:
+        memory: "4Gi"
+        cpu: "2"
+    storage:
+      size: "50Gi"
+  # - id: gateway
+  #   resources:
+  #     requests:
+  #       memory: "4Gi"
+  #       cpu: "2"
+  #     limits:
+  #       memory: "4Gi"
+  #       cpu: "2"
+  #   storage:
+  #     size: "50Gi"
+  # - id: hbase
+  #   resources:
+  #     requests:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #     limits:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #   storage:
+  #     size: "100Gi"
+  # - id: apps
+  #   resources:
+  #     requests:
+  #       memory: "4Gi"
+  #       cpu: "4"
+  #     limits:
+  #       memory: "4Gi"
+  #       cpu: "4"
+  #   storage:
+  #     size: "100Gi"
+  # - id: impala
+  #   resources:
+  #     requests:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #     limits:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #   storage:
+  #     size: "50Gi"
+  # - id: standby
+  #   resources:
+  #     requests:
+  #       memory: "4Gi"
+  #       cpu: "4"
+  #     limits:
+  #       memory: "4Gi"
+  #       cpu: "4"
+  #   storage:
+  #     size: "50Gi"
+  # - id: arbiter
+  #   resources:
+  #     requests:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #     limits:
+  #       memory: "2Gi"
+  #       cpu: "2"
+  #   storage:
+  #     size: "50Gi"
```

Comment:
remove the commented lines?

Reply:
@swamibluedata /data is required to be persisted only for HDFS. accordingly removed /data from defaultPersistDirs and added /data in persistDirs keys for primary, standby and arbiter roles. These roles use HDFS service.